driver/compile.go

```diff
@@ -536,6 +536,10 @@ func parallelizeFlowgraph(seq *ast.SequentialProc, N int, inputSortField string,
 			}
 			// put one head/tail on each parallel branch and one after the merge.
 			return buildSplitFlowgraph(seq.Procs[0:i+1], seq.Procs[i:], inputSortField, inputSortReversed, N), true
+		case *ast.DfProc:
```

Comment:
Why not handle this like UniqProc below? So that `search | df` can be parallelized. Pending all this getting subsumed by columnar, of course.